M2/Macaulay2/m2/document.m2

```diff
@@ -418,7 +418,8 @@ processSignature := (tag, fn) -> item -> (
     else if fn =!= null then fn else key;
 
     -- checking for various pieces of the synopsis item
-    isVariable   := y -> match(///\`[[:alnum:]']+\'///, y);
+    -- allow either a variable name or a visible list for rare exceptions
+    isVariable   := y -> match(///\`[[:alnum:]']+\'///, y) or match(///(\".+\")|(\(.+\))|(\[.+\])|(\{.+\})|(<\|.+\|>)///, y);
```

Comment:
What's the purpose of recognizing those other strings as variables?

Reply:
It's so that `SimpleDoc` can process something like `[a,b,c,...]` under `Inputs` here:
https://github.com/Macaulay2/M2/blob/3a7b5bfa27cc0f6631b7868128042d2950e87e2a/M2/Macaulay2/packages/Macaulay2Doc/functions/monoid-doc.m2#L23-L27